_docs/dev-setup.md

```diff
@@ -12,13 +12,14 @@ You may ask: why is the website so complicated to set up! You may have a point,
 
 ## Table of Contents
 
-Oh wow, this looks like a lot! But trust us, this only takes 15 minutes, max.
```

Comment:
Sad

Reply:
One way or another, this is gonna take people starting from scratch significantly more than 15 minutes (´。＿。｀)